kernel/shared/src/main/scala/cats/effect/kernel/Outcome.scala

```diff
@@ -129,14 +130,12 @@ object Outcome extends LowPriorityImplicits {
   implicit def monadError[F[_], E](
       implicit F: Monad[F],
       FT: Traverse[F]): MonadError[Outcome[F, E, *], E] =
-    new OutcomeApplicativeError[F, E]()(F) with MonadError[Outcome[F, E, *], E] {
+    new OutcomeApplicativeError[F, E]()(F)
+      with MonadError[Outcome[F, E, *], E]
+      with Bifunctor[λ[(α, β) => Outcome[F, α, β]]] {
```

Comment:
TIL this compiles on Dotty…

But more relevantly, there's a simpler alternative:

```suggestion
      with Bifunctor[Outcome[F, *, *]] {
```